admin/worker/run_autoscaler.go

```diff
@@ -48,20 +49,30 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 				zap.Float64("scale_threshold_percentage", scaleThreshold),
 				zap.String("project_id", targetProject.ID),
 			)
-			break
+			continue
 		}
 
-		opts := &database.UpdateProjectOptions{
-			ProdSlots: rec.RecommendedSlots,
-		}
+		updatedProject, err := w.admin.UpdateProject(ctx, targetProject, &database.UpdateProjectOptions{
+			ProdSlots:            rec.RecommendedSlots,
+			Name:                 targetProject.Name,
```

Comment:
Can you move `ProdSlots` in the same place in the list as it appears in the struct definition? It makes future refactors easier :)

Reply:
Sure 👍🏼